compatibility/bazel_tools/testing.bzl

```diff
@@ -153,19 +153,20 @@ excluded_test_tool_tests = {
     "0.0.0": [
         {
             "end": "1.0.1-snapshot.20200417.3908.1.722bac90",
-            "exclusions": ["ContractKeysIT"],
+            "exclusion-regexes": ["ContractKeys"],
```

Comment:
Can we actually be more granular now?

Reply:
I don’t remember tbh, I would just remove it and see which one fails.

Reply:
Fair enough, I'll try that.